diff --git a/dist-es/index.js b/dist-es/index.mjs
similarity index 95%
rename from dist-es/index.js
rename to dist-es/index.mjs
index 8923a9a0f7a97984eba93931fc10ea3fea809e60..bfb0de886224eec903fd2e65f61019eb337ecfcb 100644
--- a/dist-es/index.js
+++ b/dist-es/index.mjs
@@ -1,7 +1,7 @@
 import { loadConfig } from "@aws-sdk/node-config-provider";
 import { platform, release } from "os";
 import { env, versions } from "process";
-import { isCrtAvailable } from "./is-crt-available";
+import { isCrtAvailable } from "./is-crt-available.mjs";
 export const UA_APP_ID_ENV_NAME = "AWS_SDK_UA_APP_ID";
 export const UA_APP_ID_INI_NAME = "sdk-ua-app-id";
 export const defaultUserAgent = ({ serviceId, clientVersion }) => {
diff --git a/dist-es/is-crt-available.js b/dist-es/is-crt-available.mjs
similarity index 100%
rename from dist-es/is-crt-available.js
rename to dist-es/is-crt-available.mjs
diff --git a/package.json b/package.json
index fef1471213a0ed7757548775435e2d31e86006bf..c4d228d93c8a103a716bc0491d3abb512ee4e5e1 100644
--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "test": "jest"
   },
   "main": "./dist-cjs/index.js",
-  "module": "./dist-es/index.js",
+  "module": "./dist-es/index.mjs",
   "types": "./dist-types/index.d.ts",
   "author": {
     "name": "AWS SDK for JavaScript Team",